data/migratedPages.yml

```
@@ -97,6 +97,9 @@ Coding:
 Coding_style:
 - filePath: "/general/development/policies/codingstyle/index.md"
   slug: "/general/development/policies/codingstyle"
+Comment_API:
+- filePath: "/docs/apis/core/comment/index.md"
+  slug: "/docs/apis/core/comment/"
```

Slugs don't have a trailing /

```suggestion
  slug: "/docs/apis/core/comment"
```